### PR TITLE
Added Message Timer Script

### PR DESCRIPTION
**Pull request by xandao6**
I added a new script, take a look. I also updated the "usage.png" image and all other files to include this new script.

### CODE REVIEW

File: wplay/messagetimer.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	# enter message type number
+    nMessages = int(input("How many types of messages will you send? "))
+
+	# type your messages
+    messages = list()
+    for i in range(0, nMessages):
+        messages.append(str(input("Enter your message: ")))
+    n = int(input("Enter the number of messages to send: "))
+
+	#Enter the time interval of the messages, it will be sent using a random
```
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [unexpected indentation (comment)](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 7
           

Complexity increasing per file
==============================
- wplay/messagetimer.py  4
         

Clones added
============
- wplay/messagetimer.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186&bid=15883079)

File: wplay/messagetimer.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+from selenium.webdriver.common.by import By
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.common.keys import Keys
+
+
+def msgTimer(name):
+
+    target = str(name)
+
+	# enter message type number
```
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [unexpected indentation (comment)](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

File: wplay/messagetimer.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    # sends random messages multiple times
+    random.seed()
+    i = 0
+    while i <= n:
+        if not messages:
+            continue
+        message_area.send_keys(
+            messages[random.randrange(0, nMessages)] + Keys.ENTER)
+        time.sleep(random.randrange(minimumTimeInterval, maximumTimeInterval))
```
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

File: wplay/messagetimer.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+
+def msgTimer(name):
+
+    target = str(name)
+
+	# enter message type number
+    nMessages = int(input("How many types of messages will you send? "))
+
+	# type your messages
```
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [unexpected indentation (comment)](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

File: wplay/messagetimer.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        EC.presence_of_element_located((By.XPATH, xpath)))
+
+    # sends random messages multiple times
+    random.seed()
+    i = 0
+    while i <= n:
+        if not messages:
+            continue
+        message_area.send_keys(
+            messages[random.randrange(0, nMessages)] + Keys.ENTER)
```
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

File: wplay/messagetimer.py [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    nMessages = int(input("How many types of messages will you send? "))
+
+	# type your messages
+    messages = list()
+    for i in range(0, nMessages):
+        messages.append(str(input("Enter your message: ")))
+    n = int(input("Enter the number of messages to send: "))
+
+	#Enter the time interval of the messages, it will be sent using a random 
+	#interval. For fixed interval, type the same number.
```
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [unexpected indentation (comment)](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

File: wplay/messagetimer.py (hunk 1 above)
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

File: wplay/messagetimer.py (hunk 5 above)
Comment (xandao6) [reply wplay/messagetimer.py]: Is not for security/cryptographic purposes.

File: wplay/messagetimer.py (hunk 3 above)
Comment (xandao6) [reply wplay/messagetimer.py]: Is not for security/cryptographic purposes.

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- wplay/messagetimer.py  4
         

Clones added
============
- wplay/messagetimer.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186&bid=15883079)

File: wplay/messagetimer.py [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    # enter message type number
+    nMessages = int(input("How many types of messages will you send? "))
+
+    # type your messages
+    messages = list()
+    for i in range(0, nMessages):
+        messages.append(str(input("Enter your message: ")))
+    n = int(input("Enter the number of messages to send: "))
+
+    # Enter the time interval of the messages, it will be sent using a random
```
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

File: wplay/messagetimer.py [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+        EC.presence_of_element_located((By.XPATH, xpath)))
+
+    # sends random messages multiple times
+    random.seed()
+    i = 0
+    while i <= n:
+        if not messages:
+            continue
+        message_area.send_keys(
+            messages[random.randrange(0, nMessages)] + Keys.ENTER)
```
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

File: wplay/messagetimer.py [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+        time.sleep(random.randrange(minimumTimeInterval, maximumTimeInterval))
```
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- wplay/messagetimer.py  5
         

Clones added
============
- wplay/messagetimer.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186&bid=15883079)

File: wplay/messagetimer.py [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+    random.seed()
+    i = 0
+    while i < n:
+        if not messages:
+            break
+        else:
+            message_area.send_keys(
+                messages[random.randrange(0, nMessages)] + Keys.ENTER)
+            if minimumTimeInterval != maximumTimeInterval:
+                time.sleep(random.randrange(minimumTimeInterval,
```
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

File: wplay/messagetimer.py [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+    # sends random messages multiple times
+    random.seed()
+    i = 0
+    while i < n:
+        if not messages:
+            break
+        else:
+            message_area.send_keys(
+                messages[random.randrange(0, nMessages)] + Keys.ENTER)
```
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

**Comment by xandao6** [comment]
All working.

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- wplay/messagetimer.py  5
         

Clones added
============
- wplay/messagetimer.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186&bid=15883079)

File: wplay/messagetimer.py (hunk 10 above)
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

File: wplay/messagetimer.py (hunk 11 above)
Comment (rpotter12) [inline wplay/messagetimer.py]: Codacy Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=4764186)

**Pull request merged**